release/ray_release/scripts/build_pipeline.py

```diff
@@ -165,6 +165,7 @@ def main(
         or os.environ.get("BUILDKITE_SOURCE", "manual") == "schedule"
         or buildkite_branch.startswith("releases/")
     )
+    report = 1
```

Comment:
this will revert, right?

Reply:
yes, I wanted to manually run release tests and get all results. @can-anyscale said that only master and release branch will report results. So I had to temporarily change this. 
BTW, is there a reason why we don't report other branches? It'd be great to avoid manually doing this.